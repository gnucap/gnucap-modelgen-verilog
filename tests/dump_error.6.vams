
module test()
	parameter real a=1;
	parameter real a=1;
endmodule
