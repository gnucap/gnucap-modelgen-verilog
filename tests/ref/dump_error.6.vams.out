... ameter real a=1;
	parameter real a=1;
                     ^ ? already declared
